docs/Gemfile

```diff
@@ -0,0 +1,6 @@
+# If you do not have OpenSSL installed, change
+# the following line to use "http://"
+source "https://rubygems.org"
+
+# Include the tech docs gem
+gem "govuk_tech_docs", git: "https://github.com/asmega/tech-docs-gem", branch: "any-of-api-renderer"
```

Comment:
could we not move this fork to dfe repo? Otherwise, if @asmega moves on or decides to later remove this repo it will break our docs.

Reply:
Agreed. I have made the fork here: https://github.com/DFE-Digital/tech-docs-template I will add my branch there and point to it

Reply:
sorry wrong repo should be https://github.com/DFE-Digital/tech-docs-gem